release-process.rst

```diff
@@ -66,13 +89,19 @@ Patch Version Number
    -  Bug fixes and security patches only
    -  Extensions support can't be removed (like move them to PECL)
    -  Backward compatibility must be kept (internals and userland)
-   -  ABI and API compatibility must be kept (internals)
+   -  ABI and internal API compatibility must be preserved, except in the case
+      of HIGH severity security issues where no other option is possible.
 
 It is critical to understand the consequences of breaking BC, APIs or ABIs (only
 internals related). It should not be done for the sake of doing it. RFCs
 explaining the reasoning behind a breakage and the consequences along with test
 cases and patch(es) should help.
 
+If a HIGH severity security fix requires breaking the internal ABI or API, a
+proper migration path must be provided, and the impact should be minimized as
+much as possible. This should also be accompanied by additional communication
+during the release.
+
```

Comment:
- `must be provided` -> `MUST be provided`
- `the impact should be` -> `the inpact MUST be`
- `This should also be` -> `This MUST also be`

`MUST` as used in the RFC is probably more apt, as otherwise people will perhaps just ignore it.

I'm less fuzzy if these paragraphs are considered to be advice, instead of actual rules, but that isn't as clear as it ought to be.

Reply:
Yeah they are more additional advice but think it make sense to use the key words in them too as they are kind of extra clarifications that should be followed. Changed.